embulk-input-mysql/src/main/java/org/embulk/input/MySQLTimeZoneComparison.java

```diff
@@ -0,0 +1,109 @@
+package org.embulk.input;
+
+import org.embulk.spi.Exec;
+import org.slf4j.Logger;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class MySQLTimeZoneComparison
+{
+    private static final int ONE_HOUR_SEC = 3600;
+    private static final int ONE_MIN_SEC = 60;
+
+    private Connection connection;
+
+    private final Logger logger = Exec.getLogger(getClass());
+
+    public MySQLTimeZoneComparison(Connection connection)
+    {
+        this.connection = connection;
+    }
+
+    public void compareTimeZone()
+            throws SQLException
+    {
+        TimeZone serverTimeZone = null;
+        try {
+            serverTimeZone = getServerTimeZone();
+        }
+        catch (SQLException ex) {
+            logger.error(String.format(Locale.ENGLISH, "SQLException raised %s", ex.toString()));
```

Comment:
It should be `logger.warn` because it is not fatal even if timezone checking fails.